src/api/auth/url.js

```diff
@@ -0,0 +1,15 @@
+import {getBaseUrl, getInnerBaseUrl} from "../../common/environment.js";
+import {CONFIG} from "../../common/envVars.js";
+
+const baseUrl = getBaseUrl();
+const innerBaseUrl = getInnerBaseUrl();
+
+const authConfig = {
+  fullBaseUrl: `${baseUrl}/payhub-auth`,
+  fullInnerBaseUrl: `${innerBaseUrl}/p4paauth`,
+  effectiveBaseUrl: CONFIG.USE_INTERNAL_ACCESS_ENV
+      ? `${innerBaseUrl}/p4paauth`
+      : `${baseUrl}/payhub-auth`
```

Comment:
just call it baseUrl, we will not have an usage for the fullBaseUrl

```suggestion
  innerBaseUrl: `${innerBaseUrl}/p4paauth`,
  baseUrl: CONFIG.USE_INTERNAL_ACCESS_ENV
      ? `${innerBaseUrl}/p4paauth`
      : `${baseUrl}/payhub-auth`
```